pkgs/tools/graphics/vkBasalt/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, glslang
+, meson
+, ninja
+, pkg-config
+, libX11
+, spirv-headers
+, vulkan-headers
+, vkBasalt32 ? null
```

Comment:
```suggestion
, vkBasalt32
```

Reply:
Thanks, I was handling vkBasalt32 differently in my NUR repo, and it required this, but it's not necessary here.